roles/lib_utils/action_plugins/sanity_checks.py

```diff
@@ -43,6 +43,46 @@
     'openshift_prometheus_alertmanager_storage_kind',
     'openshift_prometheus_storage_kind')
 
+REMOVED_VARIABLES = (
+    # TODO(michaelgugino): Remove these in 3.11
+    ('openshift_metrics_image_prefix', 'openshift_metrics_<component>_image'),
+    ('openshift_metrics_image_version', 'openshift_metrics_<component>_image'),
+    ('openshift_grafana_proxy_image_prefix', 'openshift_grafana_proxy_image'),
+    ('openshift_grafana_proxy_image_version', 'openshift_grafana_proxy_image'),
+    ('openshift_logging_image_prefix', 'openshift_logging_image'),
```

Comment:
I don't think openshift_logging_image is anything?

Reply:
Needed for /home/mgugino/git/openshift-ansible/roles/openshift_logging/templates/jks_pod.j2